### ISSUE TITLE
adaptive adam learning rate

### ISSUE THREAD

**Issue by a-z-e-r-i-l-a** [ikostrikov/pytorch-a2c-ppo-acktr-gail#252]
I noticed that in your implementations the base learning rate of adam is decayed:

https://github.com/ikostrikov/pytorch-a2c-ppo-acktr-gail/blob/84a7582477fb0d5c82ad6d850fe476829dddd2e1/main.py#L109

Wanted to ask if you know of some papers regarding this approach for adapting the learning rate of Adam, for example in your case linearly.
